contracts/src/s-token/STokensManager.sol

```diff
@@ -53,15 +61,21 @@ contract STokensManager is
 		override
 		returns (string memory)
 	{
-		string memory _tokeUriImage = tokenUriImage[_tokenId];
+		uint256 curretnTokenId = tokenIdCounter.current();
```

Comment:
Could you add a new function like `current() view returns(uint)` to expose this value.

Reply:
I add this func